CMake/python.cmake

```diff
@@ -15,15 +15,18 @@ function(TEST_PYTHON_MODULE MODULE_NAME MODULE_PRESENT)
 endfunction()
 
 # Test python module prerequisites for a particular test script
-#   module_list - input - list of module names
+#   module_list - input - list of module names, for example "numpy;h5py"
 #   test_name - input - name of test (used for missing module message)
 #                     - use empty string to silence output
 #   add_test - output - true if all modules are present, false otherwise
 function(CHECK_PYTHON_REQS module_list test_name add_test)
+  if(NOT ${ARGC} EQUAL 3)
+    message ("Mismathced number of arguments! Wrongly called with ${ARGC} arguments.")
```

Comment:
Message should say it's expecting 3 arguments. Maybe something like:
`Expecting 3 arguments in CHECK_PYTHON_REQS.  Was called with ${ARGC} arguments.`

Reply:
Fixed